### PR TITLE
Create example k8 for a target setup

### PR DESCRIPTION
**Pull request by easternbloc**
This will need editing before it works but it gives you an idea of what you need

### CODE REVIEW

**Comment by daniel-ac-martin** [comment]
Could we remove the configmap (to simplify it) and add the secret?

**Comment by easternbloc** [comment]
@daniel-ac-martin I kinda felt like the config map did simplify it vs using more secrets. It makes it quite transparent what's getting set no?

I could just inline them i guess?

**Comment by daniel-ac-martin** [comment]
Yeah I was thinking of just inlining. Different places will have different ways of using k8s so I wanted to just present the simplest thing.

**Comment by daniel-ac-martin** [comment]
And the only secret I was thinking of was the one that you refer to, `db-creds`.

**Comment by easternbloc** [comment]
@daniel-ac-martin cool will update

**Comment by easternbloc** [comment]
@daniel-ac-martin updated

File: k8s/target.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+              value: 'target_database'
+            - name: DB_USER
+              valueFrom:
+                secretKeyRef:
+                  name: db-creds
+                  key: username
+            - name: DB_PASS
+              valueFrom:
+                secretKeyRef:
+                  name: db-creds
```
Comment (daniel-ac-martin) [inline k8s/target.yml]: We should include the secret for this explicitly. Obviously the secret wouldn't normally go in source control but this is just to demonstrate the principle.

File: k8s/target.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                  key: password
+            - name: SYNC_URL
+              value: 'http://symmetricds-target.your-namespace.svc.cluster.local/sync/target'
```
Comment (daniel-ac-martin) [inline k8s/target.yml]: `symmetricds-target.your-namespace.svc.cluster.local` can be shortened to just `symmetricds-target`.

File: k8s/target.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            - name: REGISTRATION_URL
+              value: 'http://symmetricds-source.your-namespace.svc.cluster.local/sync/source'
```
Comment (daniel-ac-martin) [inline k8s/target.yml]: This could be shortened too.

**Review by daniel-ac-martin — CHANGES_REQUESTED** [review]

**Review by daniel-ac-martin — APPROVED** [review]

**Pull request merged**